pkg/daemon/update.go

```diff
@@ -65,20 +71,27 @@ func (dn *Daemon) update(oldConfig, newConfig *mcfgv1.MachineConfig) error {
 // whatever has been written is picked up by the appropriate daemons, if
 // required. in particular, a daemon-reload and restart for any unit files
 // touched.
-func (dn *Daemon) updateFiles(oldConfig, newConfig *mcfgv1.MachineConfig) error {
+func (dn *Daemon) updateFiles(oldConfig, newConfig *mcfgv1.MachineConfig) (bool, error) {
```

Comment:
nit: I think a comment noting what the `bool` in the return would be helpful when reading the code.